src/sentry/api/endpoints/api_tokens.py

```diff
@@ -50,7 +50,7 @@ def _get_appropriate_user_id(cls, request: Request) -> str:
         """
         # Get the user id for the user that made the current request as a baseline default
         user_id = request.user.id
-        if is_active_superuser(request):
+        if superuser_has_permission(request):
```

Comment:
Hmm this is interesting for superusers. Should we allow them to fetch user tokens for another person? I believe this is only used in _admin, so not having POST will break _admin for people without superuser write.

I need to switch this check to `has_elevated_mode`, it's a bit odd when we keep touching code for diff purposes. After the feature flag is rolled out, this will only be an `is_active_staff` check so you wouldn't even need to worry about it

Reply:
if this needs to be `has_elevated_mode` then i don't need to switch it to `superuser_has_permission`! am i understanding correctly?

Reply:
I think you're both right! For the `_admin` feature, we need a way for an elevated person to grab all user tokens (i.e. the list endpoint for GET), and we don't need to worry about tokens being exposed since we don't send those anymore.

The reason we need the elevated person to grab the tokens is because we need to have the elevated person be able to delete some person's token, so we need the id of the token. The last 4 chars of the token and scopes are also returned on the GET which helps the elevated person determine which token to delete.

Whatever functionality enables `_admin` should probably be used here as well, or at least the base minimum. So if it's elevated mode I think that's sufficient